### PR TITLE
feat: add datasets navigation

### PR DESCRIPTION
**Pull request by mediremi**
https://jira.dhis2.org/browse/DHIS2-11290

### CODE REVIEW

File: src/data-workspace/data-workspace.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { useWorkflowContext } from '../workflow-context/index.js'
+import { Navigation } from './navigation.js'
 import { TitleBar } from './title-bar.js'
 
 const DataWorkspace = () => {
     const workflow = useWorkflowContext()
+    const [selectedDataSet, setSelectedDataSet] = useState(
+        workflow.dataSets.length === 1 ? workflow.dataSets[0].id : null
```
Comment (mediremi) [inline src/data-workspace/data-workspace.js:9]: From the design specs `No data set is selected by default, unless only one is available`

**Review by HendrikThePendric — CHANGES_REQUESTED** [review]
What is here is looking good, but I am pretty sure Joe mentioned we need to be able to possibly deal with more data-set-tabs than would fit in the viewport. So we need to ensure the tabs are scrollable, as [in this demo](https://ui.dhis2.nu/demo/?path=/story/navigation-tab-bar--tabs-with-scroller).

This should have been in the design specs actually.

**Comment by mediremi** [comment]
I added the `scrollable` prop to the `TabBar` and tested the changes by simulating ~50 data sets. Selection and scrolling worked fine.

**Review by HendrikThePendric — APPROVED** [review]
Yeah, I think this is fine. When I first built the TabBar it would calculate whether or not scrolling was required, but we decided to go with a simpler, more explicit, implementation in the end. I can imagine Joe will come with an additional feature request to only show the scroll buttons if they are actually required. But let's not worry about this now.

**Pull request merged**